Dependencies/build.sh

```diff
@@ -402,30 +402,49 @@ extract_sources() {
         local ext=${file##*.}
         local cmd=''
         local cmd_flags=''
-
-        case "$filetype" in
-            *application/x-tar*)
-                cmd='tar -xf';;
-            *application/x-zip*|*application/zip*)
-                cmd='unzip -q';;
+        case $(getTarget) in
+            win32)
```

Comment:
This is not correct, as this is `getHost` that is broken, not target. I do not need these hacks on macOS.